spec/smoke_tests/weird_rspec_spec.rb

```diff
@@ -33,6 +33,17 @@
 
   let(:doop) { foo; 1 }
 
+  let('foo') { 1 }
+  let('foo' \
+      'bar') { 1 }
+
+  let!('foo') { 1 }
+  let!('foo' \
+      'bar') { 1 }
+
+  let("foo#{1}") { 1 }
+  let!("foo#{1}") { 1 }
```

Comment:
maybe add some blockpasses as well

Reply:
👍 Decided to add blockpass here in https://github.com/rubocop-hq/rubocop-rspec/pull/863, which errors out as you said with blockpass, but `LetSetup` doesn't.